MoviesMVVM/MoviesMVVM/Models/TiserInfo.swift

```diff
@@ -1,15 +1,15 @@
 // TiserInfo.swift
 // Copyright © Oleg Yakovlev All rights reserved.
 
-// Модель для получения данных по тизерам
+/// Модель для получения данных по тизерам
 struct Tiser: Codable {
-    // Детали тизеров
+    /// Детали тизеров
     let results: [TiserDetail]?
 }
 
-// Модель для получения данных по тизеру
+/// Модель для получения данных по тизеру
 struct TiserDetail: Codable {
```

Comment:
```suggestion
struct TiserDetail: Codable {
```
вынеси по всем файлам 1 модель 1 файл